opensearch/Cargo.toml

```diff
@@ -29,34 +29,34 @@ rustls-tls = ["reqwest/rustls-tls"]
 aws-auth = ["aws-credential-types", "aws-sigv4", "aws-types"]
 
 [dependencies]
-base64 = "^0.21"
-bytes = "^1.0"
-dyn-clone = "~1"
+base64 = "0.21"
+bytes = "1.0"
+dyn-clone = "1"
 lazy_static = "1.4"
 percent-encoding = "2.1.0"
-reqwest = { version = "~0.11", default-features = false, features = ["gzip", "json"] }
-url = "^2.1"
-serde = { version = "~1", features = ["derive"] }
-serde_json = "~1"
-serde_with = "~3"
+reqwest = { version = "0.11", default-features = false, features = ["gzip", "json"] }
+url = "2.1"
+serde = { version = "1", features = ["derive"] }
+serde_json = "1"
+serde_with = "3"
 void = "1.0.2"
-aws-credential-types = { version = ">= 0.53", optional = true }
-aws-sigv4 = { version = ">= 0.53", optional = true }
-aws-types = { version = ">= 0.53", optional = true }
+aws-credential-types = { version = "0.53", optional = true }
+aws-sigv4 = { version = "0.53", optional = true }
+aws-types = { version = "0.53", optional = true }
```

Comment:
This restricts aws-* to `0.53.*` which causes problems for users as the aws types are part of a public api, so if user has a differing version they end up being incompatible types. 👎